src/main/python/run_trecqa.py

```diff
@@ -150,6 +161,7 @@ def get_precision(actual, predicted, k):
   parser.add_argument("-k", help="top-k passages", default=10)
   parser.add_argument("-model", help="[idf|sm]", default='idf')
   parser.add_argument('-index', help="path of the index", required=True)
+  parser.add_argument('--qa-model-file', help="the path to the model file")
```

Comment:
Just to confirm, do you want the -qa-model-file to be required argument?
This means that run_trecqa.py will not pick up the hardcoded model from castorini/models/.